index.md

```diff
@@ -10,7 +10,7 @@ layout: home
 
 This website is a result of a Service Innovation Lab project with involvement from policy practitioners from across government exploring how to improve submission analysis processes, especially when dealing with large amounts of submissions.
 
-![Data analysis phase](assets/img/analysing-data-image.png)
+<img src="assets/img/analysing-data-image.png" alt="Data analysis phase" width="50%">
```

Comment:
This is a little unusual - usually the `width` attribute on an `img` tag is set to a pixel value, not a percent value. 

I wonder if what you want here is to give it a class name or wrap it in a `div` and write some CSS to resize it, rather than writing the size directly into the element, like:

```suggestion
<div class="half-width">
  <img src="assets/img/analysing-data-image.png" alt="Data analysis phase">
</div>
```

Then in CSS somewhere:

```
.half-width {
  width: 50%;
}
```

Or, keeping in mind how it displays on mobile, maybe a multi-column flex layout?